src/index.ts

```diff
@@ -99,6 +76,7 @@ export async function handler(event: any, context: Context, cb: Callback) {
     const page = await browser.newPage();
     await page.setViewport({ width: 1200, height: 630 });
     await loadPage(page, `https://${sandboxId}.codesandbox.io/`);
+    await page.waitFor(1000);
```

Comment:
Wait a bit for fonts, images etc. to load and the sandbox to run.

Reply:
I think we should make this 2s, just to be sure.

Reply:
Done.

Reply:
Whoop!